Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM islandoracollabgroup/isle-tomcat:serverjre8
+FROM islandoracollabgroup/isle-tomcat:1.1.1
```

Comment:
I think we need to look at this a little.  Is 1.1.1 built off of master?  serverjre8 is a branch and I don't think they are equal with one another, but maybe you accounted for that?   Seems like we should merge serverjre8 back to master... not sure.